pkg/controller/controller.go

```diff
@@ -354,5 +355,22 @@ func catagoriesDisplayData(ctx context.Context, validators map[string]jvspb.Vali
 		}
 		displayData[k] = d
 	}
+
+	// In case there are additional category options available, we aim to hide the default option from users.
+	if len(displayData) > 1 {
+		delete(displayData, jvspb.DefaultJustificationCategory)
+	}
+
```

Comment:
If the default category is removed here, what happen if in the popup user selects the default category?

Reply:
If there are multiple category options("Jira Issue Key", "Github" ...), then we don't show the default category option ("Explanation") to users, users will not selects it. 
If there only one category option("Explanation"), the workflow when user select this option will be the same as before.

Reply:
I see, thx!